backend/package.json

```diff
@@ -4,8 +4,11 @@
     "build-backend": "babel ./app -d ./dist",
     "start": "nodemon --exec babel-node ./app/index.js",
     "start-backend": "NODE_ENV=production node ./dist/index.js",
-    "test": "nyc --reporter=text mocha './test/**/*.test.js'",
-    "cover": "nyc report --reporter=html --reporter=text --reporter=lcovonly --report-dir=test/coverage;"
+    "test": "jest app/",
+    "test:coverage": "yarn test --collect-coverage",
```

Comment:
O `package.json` da raiz do projeto utiliza o [Lerna](https://lerna.js.org/) para rodar os scripts comuns de todos os pacotes, então quando você roda `npm run cover`, ele roda todos os `cover` que estiverem nos pacotes. Então se for mudar, tem que mudar em todos os `package.json` dos pacotes e da raiz.